### ISSUE TITLE
Measurement of bbox_real

### ISSUE THREAD

**Issue by LeftAttention** [mks0601/3DMPPE_ROOTNET_RELEASE#37]
Congratulation for the paper. 

I am working on the pose correction due to camera parameters like camera angle, perspective transforms etc. I would like to use this algorithm. I have few questions. 

In [this](https://github.com/mks0601/3DMPPE_ROOTNET_RELEASE/blob/master/main/config.py#L38) line you have mentioned ``` bbox_real = (2000, 2000) ```. In wild image how we can measure this?

Thanks in advance.

**Comment by mks0601** [comment]
bbox_real is a constant. We use RootNet to refine that constant.

**Comment by LeftAttention** [comment]
Thanks for the quick reply.

For a straight image like [this](https://drive.google.com/file/d/154Sqxd-PdqGu-JYdS7oYfeFLBfjpDtKT/view?usp=sharing). I am getting the following 3-D pose, which is not seems to be good. Please find the 3D poses below. 

![Figure_1](https://user-images.githubusercontent.com/74410448/133390737-0fdd7a25-33c8-43b7-b1d1-0620ed2fa212.png)
![Figure_2](https://user-images.githubusercontent.com/74410448/133390742-4b97f2c1-11b7-4699-9447-9ae7218cc266.png)
![Figure_3](https://user-images.githubusercontent.com/74410448/133390746-ce87c640-90b0-437f-b247-41e4917ec51d.png)
![Figure_4](https://user-images.githubusercontent.com/74410448/133390750-0c1815af-7236-4882-9370-0e7996019cca.png)

Thanks.

**Comment by LeftAttention** [comment]
For person bounding detection I am using efficientDet. so for this case I got ``` [ 234.44705,  768.4173 , 1783.121  , 2857.573  ] ``` which are ```[xmin, ymin, xmax, ymax]```. 

And after running the rootnet with the following parameters. I got ``` tensor([[  32.3420,   32.2337, 1324.3123]], device='cuda:0') ``` which are ```  x,y: pixel, z: Foot-relative depth (mm) ```.

For the rootnet, the camera parameters are 
```python
focal = [1500, 1500] # x-axis, y-axis
princpt = [original_img_width/2, original_img_height/2] # x-axis, y-axis
```

I used the same config for the posenet demo as well.

**Comment by mks0601** [comment]
If the bounding box tightly catch human area, I guess there would be no problem.
Could you visualize the bounding box?

**Comment by LeftAttention** [comment]
Yes the bounding box is fitting almost tightly. Did I miss anything?

**Comment by LeftAttention** [comment]
Any suggestions how I need to modify the camera parameters to test wild images. Thanks.

**Comment by mks0601** [comment]
You do not have to change the camera parameters. The output 3D pose is in the focal length [1500,1500] space.
I guess there is no mistake on your side, and I think the output 3D pose isn't very wrong.
Z-axis represents distance from the camera perpendicular to the image center.
I somehow seems that the upperbody of the person in the picture is closer to the camera than the lowerbody?

**Comment by liamsun2019** [comment]
@LeftAttention  Hi, have you resolved this issue? It looks not good based on your results.

**Comment by vigneshrk29** [comment]
Hi,

Did you get it working? If yes, did you change your bounding box to [xmin,ymin, width, height]? If yes, how did you calculate height and width?

Thanks
